README.md

```diff
@@ -1,189 +1,64 @@
-| [Mobiconf](https://2018.mobiconf.org/) | **October 4-5, 2018** | 🇵🇱 Krakow, Poland |
-| [Mobile Era](https://mobileera.rocks/) | **November 1-2, 2018** | 🇳🇴 Oslo, Norway |
-| [DevFest Baltics](https://devfest.gdg.lv/) | **November 16** | 🇱🇻 Jurmala, Latvia |
-| [Functional Swift Conference 2018](http://2018.funswiftconf.com) | **December 1, 2018** | 🇺🇸 New York, NY |
-
-### More
-
-* A [Twitter list](https://twitter.com/NeoNacho/lists/cocoaconferences) with all the conferences made by [Boris Bügling](https://twitter.com/NeoNacho)
-
-If you want to add a conference to this list or edit the info, send a **pull request**, [ping me on twitter](https://twitter.com/lascorbe) or get in touch at [luis@nsspain.com](mailto:luis@nsspain.com). Thank you!
+[http://ascor.be/CocoaConferences/](http://ascor.be/CocoaConferences/)
+=========================================================
+
+List of Cocoa conferences for iOS & macOS developers.
+
+## Adding a new conference
+
+The site is generated from [this file](https://github.com/lascorbe/CocoaConferences/_data/conferences.yml); add new conferences there in YAML format.
+
+When editing an existing conference, there is no need to manually remove the corresponding date fields for past conferences; those are updated automatically. Your entry should look like this:
+
+```yaml
+- name: The name of the conference
+  link: The URL for the conference website
+  location: Text briefly describing where the conference will be held
+  start: The day the conference starts (in "year-month-day" format)
+  end: The day the conference ends (in "year-month-day" format)
+  cocoa: true or false
+  cfp:
+      link: The url at which prospective speakers my respond to the Call For Papers
+      deadline: the last day on which a talk proposal may be submitted (in "year-month-day" format)
+```
+
+### Examples
+
+The following example provides all of the necessary information. It describes a conference occuring from March 4 to March 6, 2019. It is applicable to Cocoa developers, and includes information about how to respond to the call for papers.
+
+```yaml
+- name: Example Conf 2019
+  link: https://2019.example.com/conference
+  location: 🏴 Anytown, Atlantis
+  start: 2019-03-04
+  end: 2019-03-06
+  cocoa: true
+  cfp:
+      link: https://2019.example.com/conference/cfp
+      deadline: 2019-01-31
+```
+
+This example provides some of the information, but omits the call for papers information. Perhaps it is not known, or perhaps the conference is not accepting submissions. 
+
+The `cfp` section of an item may be left incomplete, or may be omitted entirely. All other items are required.
+
+```yaml
+- name: Example Conf 2020
+  link: https://2020.example.com/conference
+  location: 🏴 Anytown, Atlantis
+  start: 2020-03-04
+  end: 2020-03-06
+  cocoa: false
+```
+
+If you get stuck, just open an issue!
+
+## Contributing
+
+1. Fork the project.
+2. Clone it locally.
+3. Edit the conferences.yml file to add your conference
```

Comment:
I think we should specify whether we'd want everyone to add a new conference at the top of the file or the bottom, not really a big issue but I think it'd be better to write it down (I'd say at the top of the file BTW: you won't have to scroll to add, and the last ones added will be there as soon as you open the file).

```suggestion
3. Edit `conferences.yml` to add your conference to the top of the file (it'll be sorted automatically by date on the auto-generated website)
```

Reply:
👍